truffle/src/com.oracle.truffle.host/src/com/oracle/truffle/host/HostTargetMappingNode.java

```diff
@@ -149,12 +150,13 @@ abstract static class SingleMappingNode extends Node {
         protected Object doDefault(Object receiver, @SuppressWarnings("unused") HostTargetMapping cachedMapping,
                         HostContext context, InteropLibrary interop, boolean checkOnly,
                         @Cached ConditionProfile acceptsProfile,
+                        @CachedLibrary(limit="3") HostLibrary library,
```

Comment:
Using cached version of `HostLibrary` because there is a node to attach to and because other nodes are `@Cached` as well.